packages/core-utils/src/bcfg.ts

```diff
@@ -1,8 +1,63 @@
+/**
+ * TypeScript typings for bcoin's BCFG config parser (https://github.com/bcoin-org/bcfg)
+ * This is NOT a complete set of typings, just what we use at Optimism at the moment.
+ * We could consider expanding this into a full set of typings in the future.
+ */
 export interface Bcfg {
+  /**
+   * Loads configuration values from the environment. Must be called before environment variables
+   * can be accessed with other methods like str(...) or uint(...).
+   *
+   * @param options Options to use when loading arguments.
+   * @param options.env Boolean, whether or not to load from process.env.
+   * @param options.argv Boolean, whether or not to load from process.argv.
+   */
   load: (options: { env?: boolean; argv?: boolean }) => void
+
+  /**
+   * Returns the environment variable with the given name and casts it as a string.
+   *
+   * @param name Name of the environment variable to query.
+   * @param defaultValue Optional default value if the variable does not exist.
+   * @returns Environment variable cast to a string.
```

Comment:
It returns the value from `argv` or from an env var

Reply:
If either is set

Reply:
Fixed